content/learn/development-guides/analysis-applications.md

```diff
@@ -1,79 +1,73 @@
+Data in the HCA Data Store can be accessed programmatically through the consumer API. This interface includes both a REST API end point and a CLI (python).
 
 [Consumer API Swagger](https://dss.integration.data.humancellatlas.org)   
 [HCA DCP CLI](https://hca.readthedocs.io/en/latest)   
 
-## Leveraging Pipelines
+## Leveraging Data Processing Pipelines
 
-**Pipelines** Our cloud-native pipelines are created using GA4GH workflow languages (CWL or WDL). On release, these pipelines can be found on Dockstore and during development can be found in GitHub. Our pipelines are modular and one can also reuse the tasks (or tools) that comprise the pipelines; we point to both pipelines and tasks below.
+The HCA DCP cloud-native data processing pipelines are created using GA4GH workflow languages (CWL or WDL). On release, these pipelines can be found on Dockstore, and during development they can be found in GitHub. The pipelines are modular, and anyone can reuse the tasks or tools that comprise them.
 
-[GitHub Repository for Pipelines (Skylab)](https://github.com/HumanCellAtlas/skylab)   
+[GitHub Repository: Pipelines (Skylab)](https://github.com/HumanCellAtlas/skylab)   
 [Pipeline Tasks](https://github.com/HumanCellAtlas/skylab/tree/master/library/tasks)   
 [Dockstore](https://dockstore.org)   
 
 ## Accessing the Matrix Service
 
-**Matrix Service API** The Matrix Service enables the ability to create matrices from underlying data in the DCP. The Matrix Service currently exposes a REST API endpoint. This can be given a manifest of files to combine, that can be polled until the service generates the requested matrix for download.
+The Matrix Service enables the creation of matrices from underlying processed data in the HCA DCP. The Matrix Service currently exposes a REST API endpoint. This interface can be given a manifest of files to combine which can be polled until the service generates the requested matrix for download.
 
 Matrix Service Description - Coming Soon    
 Matrix Service swagger - Coming Soon   
 
-## Finding Packages for Your Solution
+## Finding Registered Analysis Tools
 
-**HCA DCP Methods Registry** The Methods Registry contains domain-specific computational methodology and visualization component packages that are standardized in a way we hope enables clear communication and ease of use between the methodologists and engineers.
+The HCA DCP Analysis Tools Registry contains domain-specific computational analysis and visualization packages that are standardized to enable clear communication and ease of use. Anyone who has developed their own analysis or visualization tool that interacts with the DCP is highly encouraged to register it with the Data Portal for data hand-off or list it in the Analysis Tools Registry.
 
-[Methodology Listing](/analyze/methods/methods)   
-[Visualization Component Listing](/analyze/visualization-components)   
+[Analysis Portals Listing](https://dev.data.humancellatlas.org/analyze)
```

Comment:
use internal link: /analyze

Reply:
We've now added information about contributing packages to the Contribute section; see https://dev.data.humancellatlas.org/contribute/analysis-tools-registry
So all the wording about listing packages can refer to this page.  I'm not sure, though, what is the difference between registering a package (which seems to include the data hand-off capability) and listing a package.

Reply:
This should be [Analysis Portals Listing](/analyze). This way it will always point to the right environment instead of dev. Use this syntax for internal links.

Reply:
@jodihir 

> So all the wording about listing packages can refer to this page.

So we can replace one of the links to creating a new GitHub issue with the /contribute/analysis-tools-registry link? Which one should be replaced - Submit a Portal, Submit a Methodology (still not sure what a Methodology is), or Submit a Visualization Component?

Reply:
I think we can replace the wording in the gray boxes on all 3 of the pages; the wording: 

Are you developing a package that can consume HCA data? Please submit it for inclusion to the registry here.
can be changed to:
Are you developing a package that can consume HCA data? Please see [these instructions](https://prod.data.humancellatlas.org/contribute/analysis-tools-registry) to submit it for inclusion to the registry.

Reply:
Wait, sorry.  I was talking about the pages in the Analyze section and not this page.  But it's the same idea.  This wording on this page: "Please list your solution with the registry so our community of scientists knows about it and can easily find it."

Submit a Portal
Submit a Methodology
Submit a Visualization Component

Can be changed to:
Please list your solution with the registry so our community of scientists knows about it and can easily find it. See [these instructions](https://prod.data.humancellatlas.org/contribute/analysis-tools-registry
) to submit a portal, methodology, or visualization component